GUIDE_SORMAS2SORMAS_CERTIFICATE.md

```diff
@@ -53,23 +63,42 @@ To enable other SORMAS instances to send and receive data from this instance, th
+2. If the ``SORMAS2SORMAS_DIR`` environment variable is not available, the script will search for ``/opt/sormas2sormas`` by default. 
+If it is not found there, you will be prompted to provide the path to the *sormas2sormas* directory.
+3. If the ``SORMAS_DOMAIN_DIR`` environment variable is not available, the script will search for ``/opt/domains/sormas`` by default.
+   If it is not found there, you will be prompted to provide the path to the *sormas domain directory*.
+   >If you don't have a local sormas installation, for example you are using the docker environment, 
+   >you can specify any existing directory and after the script finishes you will find a ``sormas.properties`` file there
+   >that contains the necessary configuration that must be added to the ``sormas.properties`` file of your installation
+ 
 4. If ``sormas2sormas.truststore.p12`` is not found in the folder ``/sormas2sormas``, it will be created. 
     The truststore password may be provided in an environment variable ``SORMAS_S2S_TRUSTSTORE_PASS``.
     * If the aforementioned environment variable is not available, the truststore password will be searched in the 
     ``sormas.properties`` file.
     * If it is not found there, you will be prompted to provide the truststore password.
     * The relevant properties will be automatically set by the script in the ``sormas.properties`` file.
 5. If the server address list file ``server-list.csv`` is not found in the folder ``/sormas2sormas``, it will also be created.
-6. You will be prompted to provide the file name of the certificate to be imported. This certificate should be located
-in the ``/sormas2sormas`` folder. Please provide the name including the extension. E.g ``mycert.crt``
+6. You will be prompted to provide the *host name* of the organization that's certificate is being imported. 
+   If the certificate was generated with the `generate-cert.sh` script, the identifier can be found at the beginning of the file.
+   This certificate should be located in the ``/sormas2sormas`` folder. 
 7. After providing the requested data, the certificate will be imported to the truststore.
-8. You should have also received a CSV file with the server access data. From this file, copy the line corresponding to the
-    instance you would like to communicate with and add it to the local server address list file. This file is named
-    ``server-list.csv`` and is located in the ``/sormas2sormas`` folder. <br/>
-    *Note*: You may check that the Common Name and the Organization of the certificate match the ones corresponding to 
-    the server in the CSV file.
-9. You may now delete the ``.crt`` file.
\ No newline at end of file
+8. The content of the ``server-access-data.csv`` provided together with the certificate will be copied to the ``server-list.csv`` file.  
+9. You may now delete the ``.crt`` and ``server-access-data.csv`` files.
+
+10. *Optional for test systems or systems without a CA certificate* <br>
```

Comment:
Maybe better: `10. *Optional for test systems and other systems with self-signed ssl certificates* <br>`